aleph/tests/test_sessions_api.py

```diff
@@ -59,4 +59,4 @@ def test_password_login_post_good_email_and_password(self):
         res = self.client.post("/api/2/sessions/login", data=data)
         assert res.status_code == 200, res
         data = jwt.decode(res.json["token"], verify=False)
-        assert data["role"]["id"] == str(self.role.id), res
```

Comment:
It looked like having all the "role" fields in the token (as opposed to just the role ID) was never used anywhere so I removed it.